swagger/swagger.yaml

```diff
@@ -0,0 +1,710 @@
+        type: "string"
+        format: "uuid"
+        required: true
+      - in: "body"
+        name: "body"
+        description: "New church details"
+        required: true
+        schema:
+          $ref: "#/definitions/ChurchIn"
+      responses:
+        200:
+          description: "Church profile modified"
+          schema:
+            $ref: "#/definitions/ChurchIn"
+        404:
+          description: "Church not found"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        400:
+          description: "Invalid input"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+    delete:
+      tags:
+      - "leader"
+      summary: "Delete church"
+      description: "Delete a church from the system."
+      operationId: "deleteChurch"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "path"
+        name: "churchUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      responses:
+        204:
+          description: "Church deleted"
+        404:
+          description: "Church not found"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+
+  /church/{churchUuid}/question:
+    post:
+      tags:
+      - "leader"
+      summary: "Add question"
+      description: "Add a question to the system."
+      operationId: "addQuestion"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "path"
+        name: "churchUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      - in: "body"
+        name: "body"
+        description: "Question for the church"
+        required: true
+        schema:
+          $ref: "#/definitions/QuestionIn"
+      responses:
+        200:
+          description: "Question added"
+          schema:
+            $ref: "#/definitions/Question"
+        404:
+          description: "Church not found"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        400:
+          description: "Invalid input"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+
+  /church/{churchUuid}/question/{questionUuid}:
+    get:
+      tags:
+      - "leader"
+      summary: "Retrieve question"
+      description: "Retrieve a question for church for given UUIDs."
+      operationId: "getQuestion"
+      parameters:
+      - in: "path"
+        name: "churchUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      - in: "path"
+        name: "questionUuid"
+        type: "string"
+        format: "uuid"
+        required: true
+      responses:
+        200:
+          description: "Church question"
+          schema:
+            $ref: "#/definitions/Question"
+        404:
+          description: "Question not found"
+          schema:
+            $ref:  "#/definitions/ErrorResponse"
+    put:
+      tags:
+      - "leader"
+      summary: "Update question"
```

Comment:
Would it be easier to get rid of `Update question` and just let leaders delete and add questions?